sql/archive-tables.sql

```diff
@@ -0,0 +1,40 @@
+SET statement_timeout=0;
+
+-- Create archive_cards table if it doesn't already exist
+CREATE TABLE IF NOT EXISTS public.archive_cards (
+    id uuid NOT NULL,
+    slug character varying(255) NOT NULL,
+    type text NOT NULL,
+    active boolean NOT NULL,
+    version_major integer DEFAULT 1 NOT NULL,
+    version_minor integer DEFAULT 0 NOT NULL,
+    version_patch integer DEFAULT 0 NOT NULL,
+    version_prerelease text DEFAULT ''::text NOT NULL,
+    version_build text DEFAULT ''::text NOT NULL,
+    name text,
+    loop text,
+    tags text[] NOT NULL,
+    markers text[] NOT NULL,
+    links jsonb NOT NULL,
+    requires jsonb[] NOT NULL,
+    capabilities jsonb[] NOT NULL,
+    data jsonb NOT NULL,
+    linked_at jsonb NOT NULL,
+    created_at timestamp with time zone NOT NULL,
+    updated_at timestamp with time zone,
+    CONSTRAINT version_positive CHECK (((version_major >= 0) AND (version_minor >= 0) AND (version_patch >= 0)))
+);
+ALTER TABLE ONLY public.archive_cards ADD CONSTRAINT archive_cards_pkey PRIMARY KEY (id);
+ALTER TABLE ONLY public.archive_cards ADD CONSTRAINT archive_cards_slug_version_key UNIQUE (slug, version_major, version_minor, version_patch, version_prerelease, version_build);
```

Comment:
You should be able to create the archive tables matching the pattern of the existing tables in order to avoid any typos/accidental differences (they can still diverge after the fact, but at least the initial state will be identical). I think
```suggestion
CREATE TABLE IF NOT EXISTS public.archive_cards (
    LIKE cards INCLUDING ALL,
);
```
should work to do that

Reply:
Cool, didn't know about that 👍

Reply:
Updated in new commit